views/js/qtiCreator/editor/MathEditor.js

```diff
@@ -96,36 +101,42 @@ define(['lodash', 'jquery', 'mathJax'], function(_, $, MathJax){
     MathEditor.prototype.renderFromTex = function(){
 
         var args = _processArguments(this, arguments);
+        var _this = this;
```

Comment:
Just a notice for later, we agree on a convention for getting the parent scope into a closure. We use `var self  = this;`, except if you're in a worker...